Dockerfile

```diff
@@ -1,6 +1,21 @@
 FROM maven:3.5.2-jdk-8-alpine as maven-build
-ENV TOKENS=""
 WORKDIR /app
 COPY . .
-RUN mvn install
-CMD java -Dapi-tokens=$TOKENS -jar target/github-profile-summary-1.0-jar-with-dependencies.jar
\ No newline at end of file
+RUN mvn verify
+
+FROM openjdk:8-jre-alpine
+RUN adduser \
+ -h /var/github-summary \
+ -D -u 1000 \
+ ghsum ghsum
+
+USER ghsum
+WORKDIR /var/github-summary
+
+ENV TOKENS=""
+ENTRYPOINT ["java", "-D token=${TOCKEN}", "-jar", "github-profile-summary.jar"]
```

Comment:
`-D token=${TOCKEN}` part:
1. Spelling of `TOCKEN` is wrong.
2. I don't think there should be a space between `-D` and `token`.

Also shouldn't it be `-Dapi-tokens=${TOKENS}`?

Reply:
You're totally correct! I'm fixing this now.